community-operators/automation-broker/automationbrokeroperator.v0.2.0.clusterserviceversion.yaml

```diff
@@ -281,17 +201,23 @@ spec:
               containers:
               - name: automation-broker-operator
                 image: docker.io/automationbroker/automation-broker-operator:latest
-                imagePullPolicy: IfNotPresent
+                imagePullPolicy: Always
                 env:
                 - name: IMAGE
                   value: docker.io/ansibleplaybookbundle/origin-ansible-service-broker:v3.11
+                - name: OPERATOR_NAME
+                  value: automation-broker-operator
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
                 - name: WATCH_NAMESPACE
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
```

Comment:
This is ok because of the install mode that we select, (single ns and owned ns).

If we get a list of namespaces, we will have problems, and we will own the namespace that we are installed into. I am having a difficult time understanding why we should update this field?

Reply:
It's non-blocking, but if the opreator is ever updated to watch `multiNamespaces` or `AllNamespaces` you'll have to update this field and I wanted to draw attention to it.